packages/events/src/router/middleware/validate/index.ts

```diff
@@ -384,7 +384,7 @@ export const validateAction: MiddlewareFunction<
 
   const declaration = getCurrentEventState(event, eventConfig).declaration
 
-  if (actionType === ActionType.NOTIFY) {
+  if (actionType === ActionType.NOTIFY || actionType === ActionType.EDIT) {
```

Comment:
Is it right to validate the `Edit` action payload with `validateNotifyAction`? Is it because both can have incomplete input payload?